common/test/acceptance/tests/studio/test_studio_outline.py

```diff
@@ -1655,105 +1643,16 @@ def _verify_deprecation_warning_info(
 
     def test_no_deprecation_warning_message_present(self):
         """
-        Scenario: Verify that deprecation warning message is not shown if ORA1
-            advance modules are not present and also no ORA1 component exist in
+        Scenario: Verify that deprecation warning message is not shown if no deprecated
+            advance modules are not present and also no deprecated component exist in
             course outline.
 
         When I goto course outline
-        Then I don't see ORA1 deprecated warning
+        Then I don't see any deprecation warning
         """
         self.course_outline_page.visit()
         self.assertFalse(self.course_outline_page.deprecated_warning_visible)
 
-    def test_deprecation_warning_message_present(self):
```

Comment:
Like above, are we losing any coverage by removing instead of replacing these test cases?